### PR TITLE
Fix pre-commit blacken-docs failures

### PR DESCRIPTION
**Pull request by carmocca**
## What does this PR do?

https://github.com/PyTorchLightning/pytorch-lightning/pull/8566 was merged with pre-commit failures. cc: @Borda 

This PR attempts to fix them.

### Does your PR introduce any breaking changes? If yes, please list them.

None

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [ ] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you make sure to **update the documentation** with your changes? (if necessary)
- [n/a] Did you write any **new necessary tests**? (not for typos and docs)
- [n/a] Did you verify new and **existing tests pass** locally with your changes?
- [x] Did you list all the **breaking changes** introduced by this pull request?
- [n/a] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

### CODE REVIEW

File: pytorch_lightning/core/hooks.py [hunk 1]
```diff
@@ -410,7 +410,7 @@ def prepare_data(self):
         .. code-block:: python
 
             model.prepare_data()
-                if ddp/tpu: init()
+            initiailize_distributed()
```
Comment (carmocca) [inline pytorch_lightning/core/hooks.py:413]:
```suggestion
            initialize_distributed()
```

**Review by Borda — APPROVED** [review]
how have you changed it, manually or run another black option? :rabbit:

**Comment by carmocca** [comment]
> how have you changed it, manually or run another black option? 🐰

By resolving each of the failures (see master CI jobs).

```python
blacken-docs.............................................................Failed
- hook id: blacken-docs
- exit code: 1

docs/source/ecosystem/asr_nlp_tts.rst:245: code block parse error Cannot parse: 4:0:     @typecheck()
pytorch_lightning/profiler/__init__.py:34: code block parse error Cannot parse: 1:9: Profiler Report
pytorch_lightning/profiler/__init__.py:60: code block parse error Cannot parse: 3:0: or
pytorch_lightning/profiler/__init__.py:74: code block parse error Cannot parse: 1:9: Profiler Report
pytorch_lightning/profiler/__init__.py:100: code block parse error Cannot parse: 10:8:         return data
pytorch_lightning/profiler/__init__.py:127: code block parse error Cannot parse: 3:0: or
pytorch_lightning/profiler/__init__.py:151: code block parse error Cannot parse: 1:9: Profiler Report
pytorch_lightning/loggers/neptune.py:109: code block parse error Cannot parse: 3:4:     close_after_fit=False,
pytorch_lightning/core/hooks.py:410: code block parse error Cannot parse: 2:0:     if ddp/tpu: init()
docs/source/common/lightning_module.rst:92: code block parse error Cannot parse: 1:0: >>> import pytorch_lightning as pl
docs/source/common/lightning_module.rst:233: code block parse error Cannot parse: 12:0: <line number missing in source>
docs/source/common/lightning_module.rst:274: code block parse error Cannot parse: 24:0: <line number missing in source>
docs/source/common/lightning_module.rst:363: code block parse error Cannot parse: 12:0: <line number missing in source>
docs/source/common/lightning_module.rst:383: code block parse error Cannot parse: 24:0: <line number missing in source>
docs/source/common/lightning_module.rst:610: code block parse error Cannot parse: 6:67:     def predict_step(self, batch, batch_idx, dataloader_idx = None)
docs/source/common/lightning_module.rst:876: code block parse error Cannot parse: 1:18: def training_step(...):
docs/source/common/lightning_module.rst:887: code block parse error Cannot parse: 1:18: def training_step(...):
docs/source/common/lightning_module.rst:908: code block parse error Cannot parse: 1:18: def training_step(...):
docs/source/common/lightning_module.rst:934: code block parse error Cannot parse: 1:18: def training_step(...):
docs/source/common/lightning_module.rst:960: code block parse error Cannot parse: 1:18: def training_step(...):
docs/source/common/lightning_module.rst:971: code block parse error Cannot parse: 1:18: def training_step(...):
docs/source/common/lightning_module.rst:1034: code block parse error Cannot parse: 5:23: def on_train_epoch_end(...):
docs/source/common/lightning_module.rst:1148: code block parse error Cannot parse: 1:8: def fit(...):
docs/source/extensions/logging.rst:111: code block parse error Cannot parse: 1:18: def training_step(...):
docs/source/starter/style_guide.rst:118: code block parse error Cannot parse: 3:17:     def __init__(...):
docs/source/starter/style_guide.rst:152: code block parse error Cannot parse: 1:12: def forward(...):
docs/source/starter/style_guide.rst:165: code block parse error Cannot parse: 1:18: def training_step(...):
docs/source/starter/new-project.rst:482: code block parse error Cannot parse: 9:0: <line number missing in source>
docs/source/starter/new-project.rst:561: code block parse error Cannot parse: 3:52:     tensorboard.any_summary_writer_method_you_want())
Traceback (most recent call last):
  File "/pc/clone/_hcfpzVEQoOuvvQz-hvKAg/py_env-python3/bin/blacken-docs", line 8, in <module>
    sys.exit(main())
  File "/pc/clone/_hcfpzVEQoOuvvQz-hvKAg/py_env-python3/lib/python3.8/site-packages/blacken_docs.py", line 231, in main
    retv |= format_file(filename, black_mode, skip_errors=args.skip_errors)
  File "/pc/clone/_hcfpzVEQoOuvvQz-hvKAg/py_env-python3/lib/python3.8/site-packages/blacken_docs.py", line 187, in format_file
    new_contents, errors = format_str(contents, black_mode)
  File "/pc/clone/_hcfpzVEQoOuvvQz-hvKAg/py_env-python3/lib/python3.8/site-packages/blacken_docs.py", line 175, in format_str
    src = RST_RE.sub(_rst_match, src)
  File "/pc/clone/_hcfpzVEQoOuvvQz-hvKAg/py_env-python3/lib/python3.8/site-packages/blacken_docs.py", line 100, in _rst_match
    min_indent = min(INDENT_RE.findall(match['code']))
ValueError: min() arg is an empty sequence
docs/source/advanced/multi_gpu.rst:514: code block parse error Cannot parse: 17:5: full out = model.training_step_end(all_results)
pytorch_lightning/utilities/auto_restart.py:242: code block parse error Cannot parse: 2:18:     "batch": data returned by DataLoader
pytorch_lightning/utilities/auto_restart.py:388: code block parse error Cannot parse: 2:17:     "data": data returned by Dataset
pytorch_lightning/core/lightning.py:691: code block parse error Cannot parse: 5:4:     if optimizer_idx == 1:
pytorch_lightning/core/lightning.py:816: code block parse error Cannot parse: 4:0: <line number missing in source>
pytorch_lightning/core/lightning.py:860: code block parse error Cannot parse: 2:43: def validation_step(self, batch, batch_idx)
pytorch_lightning/core/lightning.py:893: code block parse error Cannot parse: 5:0: <line number missing in source>
pytorch_lightning/core/lightning.py:932: code block parse error Cannot parse: 25:0: <line number missing in source>
pytorch_lightning/core/lightning.py:988: code block parse error Cannot parse: 5:0: <line number missing in source>
pytorch_lightning/core/lightning.py:1035: code block parse error Cannot parse: 2:37: def test_step(self, batch, batch_idx)
pytorch_lightning/core/lightning.py:1068: code block parse error Cannot parse: 5:0: <line number missing in source>
docs/source/starter/introduction_guide.rst:474: code block parse error Cannot parse: 3:52:     tensorboard.any_summary_writer_method_you_want())
docs/source/starter/introduction_guide.rst:687: code block parse error Cannot parse: 11:4:     model.eval()
docs/source/starter/introduction_guide.rst:715: code block parse error Cannot parse: 9:0: <line number missing in source>
docs/source/advanced/advanced_gpu.rst:306: code block parse error Cannot parse: 13:62: trainer = Trainer(gpus=4, plugins='deepspeed_stage_2_offload' precision=16)
docs/source/common/early_stopping.rst:41: code block parse error Cannot parse: 3:20: def validation_step(...):
docs/source/common/trainer.rst:1313: code block parse error Cannot parse: 13:0: <line number missing in source>
```

still some errors left though

**Comment by Borda** [comment]
> By resolving each of the failures (see master CI jobs).

damm, thought we have set the pre-commit as required... sorry fo that :/

**Review by ethanwharris — APPROVED** [review]

**Review by tchaton — APPROVED** [review]
LGTM !

**Review by justusschock — APPROVED** [review]

**Pull request merged**